src/components/UserInputComponent/GDOGIDMenuComponent.js

```diff
@@ -36,8 +36,15 @@ export default function GDOGIDMenuComponent(props) {
     setOpen(false);
   };
   const handleTooltipOpen = () => {
-    setOpen(true);
+    setOpen((r) => !r);
   };
+  function setChosenGDOGIDFormikValue(gdo_gid) {
+    if (gdo_gid) {
+      formik.setFieldValue("navline.starting_gdo_gid", gdo_gid);
+    } else {
+      formik.setFieldValue("navline.starting_gdo_gid", "");
+    }
+  }
 
   useEffect(() => {
     props.setChosenGDOGIDFormikValue();
```

Comment:
```suggestion
    setChosenGDOGIDFormikValue();
```
Forgotten to removed the props.

Reply:
wups